### PR TITLE
zkApps development frameworks.

### PR DESCRIPTION
**Pull request by shimkiv**
Should be merged only after the HF related PR: https://github.com/o1-labs/docs2/pull/985 to also fix the side menu.

### CODE REVIEW

File: docs/zkapps/zkapp-development-frameworks.mdx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- Developers who wish to leverage the `modular architecture` of Protokit.
+
+Start here:
+
+- [Developer documentation](https://github.com/proto-kit)
+- [Protokit repository](https://github.com/proto-kit)
+- [Starter Kit](https://github.com/proto-kit/starter-kit)
+- [Discord](https://discord.gg/AMGnGAxsKp)
+
+## Frameworks comparison
```
Comment (garwalsh) [inline docs/zkapps/zkapp-development-frameworks.mdx:68]:
```suggestion
## Framework comparison
```

File: docs/zkapps/zkapp-development-frameworks.mdx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+o1js is also the `zkDSL` used for:
+
+- Writing general-purpose `zk circuits`.
+- Constructing `new primitives` and `data structures`.
+
+There are some key considerations when choosing to build a zkApp with o1js on Mina L1:
+
+- zkApps will be subject to protocol throughput limitations.
+- At present, zkApps that require support for multiple concurrent users require specific architecture to avoid race conditions:
+  - Where more than the `eight` on-chain field elements are required to manage state, and access to that state is not shared, the experimental `Off-chain Storage API` will suffice.
```
Comment (garwalsh) [inline docs/zkapps/zkapp-development-frameworks.mdx:40]:
```suggestion
  - Where more than the eight on-chain field elements are required to manage state, and access to that state is not shared, the experimental `Off-chain Storage API` will suffice.
```

File: docs/zkapps/zkapp-development-frameworks.mdx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+- [Discord](https://discord.gg/AMGnGAxsKp)
+
+## Frameworks comparison
+
+|| o1js SmartContract | Protokit |
+|--|--|--|
+|**Production readiness**|v1.0 released, internal audit complete, 3rd party audit in progress.|Not production ready - work in progress. Unaudited. Testnet deployments available.|
+|**Censorship resistance**|Decentralized and censorship resistant.|Censorship resistance via hybrid sequencing model.|
+|**Support for multi-user apps**|Many multi-user use cases require sophisticated architecture and are limited by L1 throughput.|Capable of handling higher throughput and multiple concurrent users, thanks to Protokit's modular sequencer.|
+|**Execution environment**|Execution off-chain, verification on-chain, transaction ordering possible on-chain.|Hybrid execution model, both on-chain (sequencer) and off-chain thanks to recursive zk-proofs, verification on-chain (MINA L1).|
```
Comment (garwalsh) [inline docs/zkapps/zkapp-development-frameworks.mdx:75]:
```suggestion
|**Execution environment**|Proving off-chain, verification on-chain, transaction ordering possible on-chain.|Hybrid execution model, both on-chain (sequencer) and off-chain thanks to recursive zk-proofs, verification on-chain (MINA L1).|
```

**Review by garwalsh — APPROVED** [review]

File: docs/zkapps/zkapp-development-frameworks.mdx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+Start here:
+
+- [Developer documentation](https://github.com/proto-kit)
+- [Protokit repository](https://github.com/proto-kit)
+- [Starter Kit](https://github.com/proto-kit/starter-kit)
+- [Discord](https://discord.gg/AMGnGAxsKp)
+
+## Framework comparison
+
+|| o1js SmartContract | Protokit |
```
Comment (stevenpack) [inline docs/zkapps/zkapp-development-frameworks.mdx:70]: The protokit part reads to me as "don't use this" whereas it should read "testnet only". Suggest reframing as:

o1js (unchanged)
v1.0 released, internal audit complete, 3rd party audit in progress.

protokit
v0.x (whatever it is), internal audit in progress, 3rd party audit not started. Testnet only.

**Pull request merged**